pulsar-client-all/pom.xml

```diff
@@ -424,10 +424,6 @@
                   <pattern>org.apache.commons</pattern>
                   <shadedPattern>org.apache.pulsar.shade.org.apache.commons</shadedPattern>
                 </relocation>
-                <relocation>
-                  <pattern>org.apache.pulsar.policies</pattern>
-                  <shadedPattern>org.apache.pulsar.shade.org.apache.pulsar.policies</shadedPattern>
-                </relocation>
```

Comment:
this change could be removed with the problem is correctly addressed in LoadReportDeserializer

Reply:
Add the excludes rule described earlier.